### ISSUE TITLE
Expand Health community

### ISSUE THREAD

**Issue by JeanneHolm** [GSA/datagov-wptheme#54]
Create a health community page with the following tabs
1. News
2. Data (display HHS federated datasets initially)
3. Community (link to Healthdata.gov)

**Comment by JeanneHolm** [comment]
Interim page available at http://next.data.gov/health

**Comment by philipashlock** [comment]
Suggest renaming the "Community" link to HealthData.gov

**Comment by dannychapman** [comment]
Agreed!

**Comment by JeanneHolm** [comment]
Completed the change in the link.

**Comment by philipashlock** [comment]
Does it really make sense to have both Data and HealthData.gov? It seems like they're redundant and like it'd make sense to just have HealthData.gov

**Comment by JeanneHolm** [comment]
HealthData.gov is the community space for the HHS focused initiatives around the use of health data, and connects to some additional agencies.  

The intent of the Health topic at Data.gov (Health.Data.gov) is to have the federated health datasets from across all agencies at Data.gov and to provide a way to see that topic from the overall U.S. Government perspective in coordination with other important open data topics. Earlier we had removed it from Data.gov, but the other topics on Data.gov need to be able to point to and integrate health datasets into their topics as well.

**Comment by cew821** [comment]
I feel like it is pretty redundant to create a new topic / content site focused on government health data unless there is a really good reason. It's hard to imagine data.gov/health having content that would be better, newer or more relevant than healthdata.gov. And if it did, I feel like healthdata.gov would try to repost it anyways.

I agree that we need to provide a way to navigate people to health datasets from data.gov, but potentially we could do that by pointing our topic page to healthdata.gov?

**Comment by philipashlock** [comment]
Currently the data link on http://next.data.gov/health/ just points to the HHS datasets on catalog.data.gov. If it included federated datasets across all agencies it would exist as a curated topic on catalog.data.gov, but there doesn't seem to be one for Health. Until such curation exists, I suggest we remove the Data link. 

Also, currently http://data.gov/health and http://health.data.gov both point to healthdata.gov 

It's also worth noting that a large portion of the blog posts on next.data.gov/health seem to be cross posts from other sites.

**Comment by JeanneHolm** [comment]
We tried that already, and there were a lot of usability issues.  Some of these have been:
  --In clicking on Health on Data.gov, it goes today to HealthData.gov and it is a very different look and feel, structure of navigation, and presentation of the data
  --The links from other topics to health data and content break frequently when the sites are fully separated

So the plan forward it to provide a slim version of updates (feeds received from HHS blogs and @Healthdatagov Twitter and updates cross-posted from other topic areas).  The Data tab links to a collection of data that is curated for health, but includes more sources than HealthData.gov does. These data are presented the same way all other data on Data.gov are via the CKAN catalog.  The community link then takes you to HealthData.gov, but only after bringing users through the same structure, functionality, search, and data access that they expect.

**Comment by philipashlock** [comment]
@JeanneHolm the Data link on http://next.data.gov/health/ just points to the HHS datasets on catalog.data.gov - http://catalog.data.gov/organization/hhs-gov

I suggest we remove the Data link until we have a collection of data this is curated for health from more sources than healthdata.gov does.

**Comment by JeanneHolm** [comment]
Each topic needs to have at a minimum the data tab and news. People go to the topic to find the datasets that are applicable there.  We should keep the tab and I am working on the curation of that list with HHS so that it is inclusive of a broader group of agencies, as well as potentially some cities and states from Cities.Data.gov and States.Data.gov.

**Comment by Raseman** [comment]
If we do keep some kind of link to the health data in the Data.gov catalog,
there should be a prominent way when you click to the Data tab to see the
option to click through to the data catalog on healthdata.gov which
explains the difference between the two catalogs concisely.

---

Sophie Raseman
+1 (203) 500-7090
raseman@gmail.com

**Comment by philipashlock** [comment]
I agree we should provide clarity about the distinction between this section, especially the Data link, and healthdata.gov. Perhaps that that could be part of the introduction of the Health topic or the Data link could first be a page explaining this distinction before sending you to catalog.data.gov. I'll leave it to you to decide @JeanneHolm and we can always create a new or more specific ticket as needed, but otherwise I think we could call this complete.

**Comment by philipashlock** [comment]
Also regarding content, see #267

**Comment by DamonDavis** [comment]
Not sure what it's worth, but I'm chiming in to say I can see how there is utility in Data.gov having a health domain separate from HealthData.gov.  If I were new to the open data space and I discovered Data.gov and its various domains, it would be cool to quickly find health data from HHS as well as mabye the VA health system.  Department of Ed might have school nurse health data.  Dept of Labor's has occupational health and safety data?  Agriculture has health data. HealthData.gov would like to have all of those resources, but we don't yet.

**Comment by philipashlock** [comment]
At this point it looks like the only thing we're missing is the Health topic in the data catalog, but the original issue as described here seems to have been resolved. I'll create a new issue for curating the datasets.
